src/templates.rs

```diff
@@ -295,89 +295,3 @@ pub fn choose_or_get_template(
 
     Ok(chosen_template_name)
 }
-
-#[cfg(test)]
-mod tests {
-    // Note this useful idiom: importing names from outer (for mod tests) scope.
-    use super::*;
-    use handlebars::Handlebars;
-    use serde_json::{Map, Value};
-
-    #[test]
-    fn test_get_scope_open_and_close_char_indexes() {
-        let text = String::from("const s = {};");
-        let scope_opener = String::from("const s = {");
-
-        let (scope_opener_index, scope_close_index) =
-            get_scope_open_and_close_char_indexes(&text, &scope_opener).unwrap();
-
-        assert_eq!(scope_opener_index, 10);
-        assert_eq!(scope_close_index, 11);
-    }
-
-    #[test]
-    fn test_merge_match_scope() {
```

Comment:
I agree the test above is duplicated but this one was added recently and looks like it has differences to the other `test_merge_match_scope`. It would be worth checking with @guillemcordoba on this test removal I think

Reply:
It looks plausible that the differences are cosmetic and the tests are actually checking the same thing but I believe this was added to close a bug recently so I'd like to check